### PR TITLE
When using glutin, set the mouse cursor

### PR DESCRIPTION
**Pull request by bjwbell**
r? @glennw

### CODE REVIEW

**Comment by glennw** [comment]
@bjwbell Looks good, thanks!

Will hold off on merging just for now, as it looks like the rust upgrade PR is nearly ready to go. Once that lands, I'll merge this.

**Comment by glennw** [comment]
@bjwbell I just checked the mac implementation of glutin, and it calls unimplemented!() for set_cursor, which will cause crashes on mac unfortunately. This should probably wait until glutin is changed to implement set_cursor for mac, or the unimplemented() is removed, so that it does nothing on mac. Sorry for the delay!

**Comment by bjwbell** [comment]
Hi @glennw I modified it to only call glutin's set_cursor on Linux. Will that work?

Thanks

**Comment by glennw** [comment]
@bjwbell - Could you just add a small TODO comment in the code to enable on mac when supported in glutin, squash the commits and then I'll merge it. Thanks! :)

**Comment by bjwbell** [comment]
@glennw Sure, I'll do it in the morning.

**Comment by bjwbell** [comment]
@glennw Finished, you should be able to merge. Thanks & have a good weekend!

File: ports/glutin/window.rs [hunk 1]
```diff
@@ -491,8 +491,56 @@ impl WindowMethods for Window {
     fn load_end(&self) {
     }
 
+    // TODO: Enable on macos when supported by glutin.
+    #[cfg(target_os="macos")]
+    fn set_cursor(&self, _: Cursor) {
+    }
+
```
Comment (Manishearth) [inline ports/glutin/window.rs]: Nit: Extra whitespace

File: ports/glutin/window.rs [hunk 2]
```diff
@@ -491,8 +491,56 @@ impl WindowMethods for Window {
...
 
+    // TODO: Enable on macos when supported by glutin.
+    #[cfg(target_os="macos")]
+    fn set_cursor(&self, _: Cursor) {
+    }
+    
+    #[cfg(target_os="android")]
     fn set_cursor(&self, _: Cursor) {
     }
+
```
Comment (Manishearth) [inline ports/glutin/window.rs]: Nit: Extra whitespace

**Comment by Ms2ger** [comment]
@jdm: and we've got our first unused import! Want to fix?

**Comment by bjwbell** [comment]
@Ms2ger I didn't even notice that for compiling on macos or android, sry.

**Pull request closed without merging**